ws-client/src/client.rs

```diff
@@ -269,7 +269,7 @@ impl<'a> WsClientBuilder<'a> {
 
 		let (sender, receiver) = builder.build().await.map_err(|e| Error::Transport(Box::new(e)))?;
 
-		async_std::task::spawn(async move {
+		tokio::spawn(async move {
```

Comment:
the only downside with this is that `tokio` must be initialized for this to work,

Reply:
It's an `async` function, thus it's unlikely that it will be executed without runtime.

Reply:
true, we could add some docs that it must be spawned on `tokio`

Reply:
Given it changes the current behaviour I think we should document this.